.github/workflows/iroha2-dev-pr-heavy.yml

```diff
@@ -1,7 +1,7 @@
 name: I2::Dev::Tests (Heavy)
 
 on:
-  pull_request:
+  push:
```

Comment:
Hopefully it is temporary. It might be a good idea to move both the docker test and the benchmarks into the workflow for merging from `iroha2-dev` to `iroha2`. I'm doing it until we fix the docker test (or move it to a kubernetes test).